{{cookiecutter.project_name}}/Makefile

```diff
@@ -121,6 +121,7 @@ README-pypi.html: README.rst
 	$(RST2HTML) README.rst README-pypi.html
 README.rst: README.md
 	pandoc -f markdown_github -t rst -o README.rst README.md
+	$(PYTHON) setup.py check -rsm
```

Comment:
If the `README.md` contains something invalid, the `$ make readme` target will fail on its first run, but on its second run it will say "nothing needs to be done". Perhaps this line could be followed by something like `... || { rm README.rst; echo "README.md is invalid" }`?